### DISCUSSION TITLE
Updating the Wiki Getting Started page

### DISCUSSION THREAD

**Discussion by nigelkennington** [tsMezotic/MonoGame.ImGuiNet#6] [General]
Hi,

I've just imported the library and translated the code from the base dearimgui page to do the "My First Tool" example:
![image](https://github.com/Mezo-hx/MonoGame.ImGuiNet/assets/14282301/22ff9c74-e353-4fb5-8ec5-b9e678f97182)

I'd be happy to write this up as a tutorial on the Getting Started wiki page if you would give write access to it.

Thanks,

Nigel.

**Comment by tsMezotic** [comment]
Hey! I would absolutely love that since I won't have any time to write something currently. 

You should have an invitation in your email!

**Reply by nigelkennington** [reply]
Thanks! Done.

Added two pages to the wiki and a link to the readme.md file to reference them. They're a bit rough, but let me know if I've got anything wrong!
